CHANGELOG.md

```diff
@@ -3,8 +3,18 @@
   - Bugs Fixed
 
 ## Next release
+  - Features Added
+  - Bugs Fixed
+
+## 0.10.0
 
   - Features Added
+    - added support for EnsRegistrar
```

Comment:
I would say add support for EnsRegistrar generic scheme or something like that

Reply:
done